napari/layers/tracks/_track_utils.py

```diff
@@ -190,7 +183,7 @@ def properties(self, properties: Dict[str, Array]):
         self.features = properties
 
     @property
-    def graph(self) -> Dict[int, Union[int, List[int]]]:
+    def graph(self) -> Optional[Dict[int, List[int]]]:
```

Comment:
Have you checked whether an empty dict would still be working / be better ?
I guess `{}` is falsish , like None but might make typing simpler ?

(not for this PR as you seem to try to keep the API almost identical maybe)

Reply:
I like this idea, and started making the change, then realised there's quite a few places where we could remove `Optional` in the tracks API and use empty dicts or arrays, so I think that's best left to a separate PR.